src/drivers/boards/aerofc-v1/aerofc_usb.c

```diff
@@ -78,13 +78,6 @@
 
 __EXPORT void stm32_usbinitialize(void)
```

Comment:
If there is no USB on the board, you have a great opportunity that can realized by running menuconfig and turning off all the the USB support on the System Type level and the OS device drivers level.

1) You will be able to get rid of the file src/drivers/boards/aerofc-v1/aerofc_usb.c
2) Free a bunch of flash and RAM
3) Use will be able to use the full 2M on the the F427. (USB pin errata will no longer apply) by changing the linker map.